build.gradle

```diff
@@ -10,14 +10,14 @@
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-	ext.kotlin_version = "1.5.30"
+	ext.kotlin_version = "1.6.10"
 	repositories {
 		google()
 		mavenCentral()
 	}
 	dependencies {
-		classpath 'com.android.tools:r8:3.0.69'
-		classpath 'com.android.tools.build:gradle:7.0.3'
+		classpath 'com.android.tools:r8:3.0.73'
```

Comment:
I think we included this version due to a bug in R8 that made problems with the reproducible builds. Could you check with @goebelUB whether this version contains the fix?

Reply:
3.0.69 included the fix, so I would hope that it's still there in 3.0.73 :)

Sidenote: we only included the r8 dependency to manually override whatever r8 version the Android toolchain ships with. If at some point the toolchain comes with a fixed version as well, we could remove the explicit r8 dependency alltogether.

Reply:
Perfect, thanks :smiley: